clusterloader/cluster_loader.go

```diff
@@ -93,22 +107,46 @@ var _ = framework.KubeDescribe("Cluster Loader [Feature:ManualPerformance]", fun
 	})
 })
 
+// appendUnique appends new namespace pointers to a slice
+func appendUnique(allNS []*v1.Namespace, newNS *v1.Namespace) []*v1.Namespace {
+	for _, NS := range allNS {
+		if NS.Name == newNS.Name {
+			return allNS
+		}
+	}
+	return append(allNS, newNS)
+}
+
 // getTuningSet matches the name of the tuning set defined in the project and returns a pointer to the set
-func getTuningSet(tuningSets []clusterloaderframework.TuningSetType, podTuning string) (tuning *clusterloaderframework.TuningSetType) {
-	if podTuning != "" {
+func getTuningSet(tuningSets []clusterloaderframework.TuningSetType, tuningName string) (tuning *clusterloaderframework.TuningSetType) {
```

Comment:
In general I'm against of having "Type" in the type name definition - we already know this is type, so this is useless.

That said, once we switch it to "TunningSet", I suggest introducing "TunningSetList" (or sth like that) type and making this a method on it.